packages/website/pages/login.js

```diff
@@ -42,17 +42,17 @@ export default function Login() {
     }
   }
   return (
-    <main className="flex-auto bg-nsorange w-100">
-      <div className="mw9 center pv5 mtauto">
+    <main className="flex-auto bg-nsorange w-full">
+      <div className="max-w-7xl mx-auto py-16 mtauto">
```

Comment:
mt-auto

Reply:
mt-auto isn't a thing? I don't see what you are asking/referencing.

Reply:
**discussion:** The div on 46 in both before and after has the class "mtauto"; the TW class is: "mt-auto"

https://tailwindcss.com/docs/margin